frontend/freebase2wikidata.js

```diff
@@ -318,6 +367,23 @@ $(function() {
       });
     })();
 
+     // Add create Primary Sources nav list button
+    (function createPSNavList() {
```

Comment:
I think this step should be avoided, since it requires an extra click.
Less clicks, better user experience.
The user knows he/she uses the primary sources tool, so the property list should automatically show up.
Ideally **above** the _tools_ section.